package.json

```diff
@@ -45,10 +45,14 @@
   "license": "ISC",
   "dependencies": {
     "babel-plugin-transform-class-properties": "^6.24.1",
+    "formik": "^2.2.9",
```

Comment:
looks like you are no where using this dependency. If yes, remove it. Also, consider adding an associated linting rule.